docs/reference/gadget-helper-api.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Gadget helper API'
+weight: 10
+description: 'Reference documentation for the gadget helper API'
+---
+
+The gadget helper API is a set of features exposed by Inspektor Gadget that is accessible from eBPF code.
+
+## Container enrichment
+
+To make use of container enrichment, gadgets must include the relevant header:
+```
+#include <gadget/mntns_filter.h>
```

Comment:
Unfortunately it doesn't work, please check https://github.com/inspektor-gadget/inspektor-gadget/pull/2227.